components/back-button/back-button.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./back-button.css";
+import SVG from "react-inlinesvg";
+import leftArrow from "../../public/chevron-left.svg";
+
```

Comment:
Add JSDoc comments to add documentation and VScode intellisence to the component.

Reply:
JSDoc comments are still missing.